apps/dash-cytoscape/README.md

```diff
@@ -0,0 +1,56 @@
+# Dash Cytoscape Phylogeny
+
+## About this app
+
+This is a demo of the Dash interactive Python framework developed by Plotly.
+
+
+
+## How to run this app
+
+(The following instructions apply to Windows command line.)
```

Comment:
I think these instructions apply to Unix machines (given the direction of the slashes when navigating to the app directory)!